.github/workflows/rebuildDependencies.yml

```diff
@@ -0,0 +1,78 @@
+name: VCMI - vcpkg dependencies
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+  schedule:
+    - cron: '0 2 * * 0' #every sunday 02:00
+  workflow_dispatch:
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - platform: msvc-x64
+            triplet: x64-windows
+          - platform: msvc-x86
+            triplet: x86-windows
+
+    runs-on: windows-latest
+    defaults:
+      run:
+        shell: bash
+
+    steps:
+    - name: Checkout vcpkg repository
```

Comment:
Windows runner has vcpkg already. If the version doesn't suit us, I guess you could simply git pull from the directory.

Vcpkg (build from commit 3a747a4fc)

Environment variables: VCPKG_INSTALLATION_ROOT

Reply:
Haven't checked it, was mostly following existing guide from readme.md in this repo.